test/fixtures/grayscale.json

```diff
@@ -1,62 +1,76 @@
 {
   "type": "root",
-  "children": [{
```

Comment:
Aside from indentation, I do not see any obvious changes. Correct?

Reply:
There's a lot of noise (formatting) but the real change starts at line 232.